book/05-more-rmarkdown.Rmd

```diff
@@ -164,8 +164,7 @@ Firstly lets make sure we have our data properly filtered. We are going to filte
 
 ```{r echo = F, comment=""}
 text <- c("```{r}",
-          "df2 <- df %>% filter(Airline == 'Jetstar' | Airline == 'Qantas')",
-	  "                     Route != 'All Ports-All Ports')",
+          "df2 <- df %>% filter( (Airline == 'Jetstar' | Airline == 'Qantas') & Route != 'All Ports-All Ports')",
```

Comment:
Thanks for picking this up, it was a bug. I'm a little unsure if it meant to be AND (`&`) or simply comma? In this case I'm doing two separate filtering, one for airlines and one for route. I think what you have there is fine, but just a little unsure

Reply:
Modified the filter syntax, replaced & with ",".